Lib/test/test_unicode.py

```diff
@@ -2830,6 +2830,28 @@ def test_asucs4(self):
             self.assertEqual(unicode_asucs4(s, len(s), True), s+'\0')
             self.assertEqual(unicode_asucs4(s, len(s), False), s+'\uffff')
 
+    # Test PyUnicode_GetUTF8Buffer()
+    @support.cpython_only
+    def test_getutf8buffer(self):
+        from _testcapi import unicode_getutf8buffer, unicode_test_getutf8buffer
+
+        # Run tests wrtten in C.  Raise an error when test failed.
+        unicode_test_getutf8buffer()
+
+        ascii_ = "foo"
```

Comment:
I suggest to rename ascii_ to asciistr or ascii_str. The "_" suffix looks strange :-)